app/main.py

```diff
@@ -2,5 +2,13 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache = dict()
+
+    def wraper(*args, **kwargs) -> int:
+        if args in cache:
```

Comment:
Using `args` directly as a key in the cache dictionary is fine since tuples are hashable. However, if you plan to use `kwargs` as well, you need to convert them to a hashable type before using them as a key.